docs/docs/frigate/hardware.md

```diff
@@ -107,6 +107,10 @@ Frigate supports hardware video processing on all Rockchip boards. However, hard
 
 The inference time of a rk3588 with all 3 cores enabled is typically 25-30 ms for yolo-nas s.
 
+#### Hailo-8l PCIe
+
+Frigate supports the Hailo-8l M.2 card on any harware but currently it is only tested on the Raspberry pi5 PCIe hat from the AI kit.
```

Comment:
It might be nice for users to see a typical inference time here like the other hardware above has.

Reply:
How do I find that out?

Reply:
Load the frigate UI and go to the system page